programs/test/benchmark.c

```diff
@@ -643,6 +627,10 @@ int main( int argc, char *argv[] )
     memset( buf, 0xAA, sizeof( buf ) );
     memset( tmp, 0xBB, sizeof( tmp ) );
 
+    /* Avoid "unused static function" warning in configurations without
+     * symmetric crypto. */
+    (void) mbedtls_timing_hardclock( );
```

Comment:
Minor: there's no point in calling the function. `(void) mbedtls_timing_hardclock;` would both silence the unused-function warning and allow the compiler to not generate code for the function.

Reply:
By the way, this bug fix doesn't apply to 2.28: there, `mbedtls_timing_hardclock` is an exported function from `timing.o`.